app/controllers/application_controller.rb

```diff
@@ -1,3 +1,4 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+  #protect_from_forgery with: :null_session
```

Comment:
Remove commented out lines (or if you need to keep it, write beside it that its for archiving purposes)